### ISSUE TITLE
Creating our website

### ISSUE THREAD

**Issue by NataliaLKB** [node-girls/hackday#9]
As stated in #7 , @nikhilaravi @iteles and I are all working on the website today.

@heron2014 came up with a beautiful design which is here: https://github.com/node-girls/start-here/issues/7. Most of the pictures are in that issue and/or in the google drive.
# What do we want to have on the website?
- Contact us
- Meet the team
- What are we? Why?
- Resources
- Events/Calendar
- Twitter/News
- Code of Conduct
- Footer
  - Twitter/Email/Github
  - Sponsor Logos
### Maybes / second iteration:
- Subscribe
- Support us
### On the landing:
- Why
- Upcoming Events
- News panel -> Twitter feed
- Footer

**Comment by NataliaLKB** [comment]
@nikhilaravi , @iteles and I hope to finish what we need to for this to go live by EOD Saturday the 6th.

**Comment by nikhilaravi** [comment]
yep i've done the team page, will make a PR and add instructions on how everyone should add their images/github links.

**Comment by NataliaLKB** [comment]
Great thanks @nikhilaravi
